pkg/bmc/idrac_virtualmedia.go

```diff
@@ -86,3 +88,53 @@ func (a *redfishiDracVirtualMediaAccessDetails) RAIDInterface() string {
 func (a *redfishiDracVirtualMediaAccessDetails) VendorInterface() string {
 	return "no-vendor"
 }
+
+func (a *redfishiDracVirtualMediaAccessDetails) ValidateBootInterface(bmcCreds Credentials) (bool, error) {
+	var hasVirtualMedia, err = a.validateVirtualMedia(bmcCreds)
+	return hasVirtualMedia, err
+}
+
+func (a *redfishiDracVirtualMediaAccessDetails) validateVirtualMedia(bmcCreds Credentials) (bool, error) {
+	bmcConfig := gofish.ClientConfig{
+		Endpoint: getRedfishAddress(a.bmcType, a.host),
+		Username: bmcCreds.Username,
+		Password: bmcCreds.Password,
+		Insecure: !a.disableCertificateVerification,
+	}
+
+	client, err := gofish.Connect(bmcConfig)
+	if err != nil {
+		panic(err)
```

Comment:
should it panic, or simply return that error (wrapped for context if necessary)

Reply:
It should definitely not panic.